src/libraries/System.Net.Http.Json/ref/System.Net.Http.Json.cs

```diff
@@ -0,0 +1,43 @@
+        public static System.Threading.Tasks.Task<System.Net.Http.HttpResponseMessage> PutAsJsonAsync<T>(this System.Net.Http.HttpClient client, string requestUri, T value, System.Text.Json.JsonSerializerOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public static System.Threading.Tasks.Task<System.Net.Http.HttpResponseMessage> PutAsJsonAsync<T>(this System.Net.Http.HttpClient client, System.Uri requestUri, T value, System.Text.Json.JsonSerializerOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+    }
+    public static partial class HttpContentJsonExtensions
+    {
+        public static System.Threading.Tasks.Task<object> ReadFromJsonAsync(this System.Net.Http.HttpContent content, System.Type type, System.Text.Json.JsonSerializerOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public static System.Threading.Tasks.Task<T> ReadFromJsonAsync<T>(this System.Net.Http.HttpContent content, System.Text.Json.JsonSerializerOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+    }
+    public partial class JsonContent : System.Net.Http.HttpContent
+    {
+        public JsonContent(System.Type type, object? value, System.Net.Http.Headers.MediaTypeHeaderValue mediaType, System.Text.Json.JsonSerializerOptions options = null) { }
+        public JsonContent(System.Type type, object? value, string mediaType, System.Text.Json.JsonSerializerOptions options = null) { }
+        public JsonContent(System.Type type, object? value, System.Text.Json.JsonSerializerOptions options = null) { }
+        public System.Type ObjectType { get { throw null; } }
+        public object? Value { get { throw null; } }
+        public static System.Net.Http.Json.JsonContent Create<T>(T value, System.Net.Http.Headers.MediaTypeHeaderValue mediaType, System.Text.Json.JsonSerializerOptions options = null) { throw null; }
+        public static System.Net.Http.Json.JsonContent Create<T>(T value, string mediaType, System.Text.Json.JsonSerializerOptions options = null) { throw null; }
+        public static System.Net.Http.Json.JsonContent Create<T>(T value, System.Text.Json.JsonSerializerOptions options = null) { throw null; }
+        protected override System.Threading.Tasks.Task SerializeToStreamAsync(System.IO.Stream stream, System.Net.TransportContext context) { throw null; }
```

Comment:
No cancellation token?

Reply:
It's an override - this would have to be plumbed through a few layers to add that. (My first time noticing this as well).

Reply:
There's an overload in .NET 5 that takes a CancellationToken.  We'll want to override that as well when this ref / implementation are built for .NET 5.